packages/nakama-js-protobuf/package.json

```diff
@@ -3,7 +3,14 @@
     "version": "1.1.0",
     "description": "Websocket adapter adding protocol buffer support to the Nakama Javascript client.",
     "main": "dist/nakama-js-protobuf.cjs.js",
-    "module": "dist/nakama-js-protobuf.esm.js",
+    "module": "dist/nakama-js-protobuf.esm.mjs",
+    "exports": {
+        "./package.json": "./package.json",
```

Comment:
Note: Exporting `package.json` is a common convention as libraries/bundlers frequently use it. See https://github.com/nodejs/modules/issues/445